### PR TITLE
Details popup

### PR DESCRIPTION
**Pull request by JustoRago**
Made the main page's projects section dynamic, pulling project information from a javascript array. Made similarly dynamic modals.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -370,6 +368,78 @@ h4 {
...
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  margin-left: 1.6rem;
+  margin-top: 2rem;
+  min-width: 0;
+  min-height: 0;
+  width: 90vw;
+  height: 100vh;
+  align-self: center;
+  overflow: hidden;
+}
```
Comment (MahmoudBakr23) [inline styles.css]: - [ ] Well done styling the popup window so far. However, you may have to write some more CSS code to shape the widths and heights of it across multiple screen sizes. Please take a look at the two images down below.
**Desktop**
Screenshot (2)
**Mobile**
Screenshot (1)

- [ ] Kindly adjust your design to make it fit the multiple sizes of a screen 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @asdt560 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done with following the Figma design
- Great work with javascript
- Descriptive PR and Readme file
- Correct Linters

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process improves the code you wrote, not criticizes it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention me @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 2]
```diff
@@ -6,4 +6,120 @@ function menuOpen() {
...
+    </ul>
+    <div class="popupbuttoncontainer">
+    <button class="project-button popupbutton">See Live<img src="./images/Live.svg" alt="Live"></button>
+    <button class="project-button popupbutton">See Source <img src="./images/Normal-Button/Tertiary/Icons/Vector.png" alt="Github"></button>
+    <div>
+  </div>`;
+    }
+  });
+}
+
```
Comment (tresorsawasawa) [inline script.js]: - [ ] Good job doing some of the changes required by the previous reviewer. But as he mentioned,  kindly make sure your popup design matches the one from your chosen Figma template. Right now the **ClosePopup*, **See Live**, and **See Source** buttons disappear on some desktop screen sizes like on the image below:

### Your Design
Screenshot from 2022-09-07 02-59-23 
Screenshot from 2022-09-07 03-07-29 |

#### Figma Design
Screenshot from 2022-09-07 03-00-36

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @asdt560 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @asdt560 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code.

- n/a

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**